src/test/com/yetanalytics/datasim/xapi/statement_test.clj

```diff
@@ -29,7 +29,7 @@
           valid-args {:input input
                       :iri-map iri-map
                       :activities activities
-                      :actor (-> input :personae :member first)
+                      :actor (-> input :personae :member first (dissoc :role))
```

Comment:
what's the rationale dissoc-ing this here, just to keep the output the same?

Reply:
If we do not dissoc `:role`, the statements will fail validation against `::xs/statement`, since roles are obviously not part of actual statements.